.github/workflows/integration-tests.yaml

```
@@ -32,14 +32,8 @@ jobs:
           set-safe-directory: true
           path: go/src/github.com/akash-network/provider
-      - name: Detect required Go version
-        working-directory: "${{ env.GOPATH }}/src/github.com/akash-network/provider"
-        run: |
-          toolchain=$(./script/tools.sh gotoolchain | sed 's/go*//')
-          echo "GOVERSION=${toolchain}" >> $GITHUB_ENV
-      - uses: actions/setup-go@v5
-        with:
-          go-version: "${{ env.GOVERSION }}"
+      - name: Setup environment
+        uses: ./.github/actions/setup-ubuntu
```

actionlint (1.7.8)

description is required in metadata of "setup-ubuntu" action at "/home/jailuser/git/.github/actions/setup-ubuntu/action.yaml"

(action)